app/Http/Controllers/TestController.php

```diff
@@ -0,0 +1,11 @@
+<?php
+namespace App\Http\Controllers;
+
+class TestController extends Controller{
+	public function test(){
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->